test/suites/auth.sh

```diff
@@ -1221,13 +1221,28 @@ entities_enrichment_with_entitlements() {
   # Storage pool
   lxc auth group permission add test-group storage_pool "${pool_name}" can_edit
   lxc auth group permission add test-group storage_pool "${pool_name}" can_delete
+
+  # Test with one pool
   [ "$(lxc_remote query "oidc:/1.0/storage-pools/${pool_name}?with-access-entitlements=can_edit,can_delete" | jq -r '.access_entitlements | sort | @csv')" = '"can_delete","can_edit"' ]
+
+  # Test multiple pools
+  lxc storage create foo dir
+  lxc auth group permission add test-group storage_pool foo can_edit
   all_storage_pools=$(lxc_remote query "oidc:/1.0/storage-pools?recursion=1&with-access-entitlements=can_edit,can_delete")
-  if ! jq -e -r --arg pool_name "$pool_name" '.[] | select(.name == $pool_name) | .access_entitlements | sort | @csv == "\"can_delete\",\"can_edit\""' <<< "$all_storage_pools";  then
-    echo "Second check failed (or pool not found)!" >&2
-    false
-  fi
+  jq -e -r --arg pool_name "$pool_name" '
```

Comment:
Please can you use long form flags where they exist, for documentation reasons.

Also can we have a comment explaining this complex command.

Reply:
Gabriel added these checks a while ago and this is (almost) a copy/paste. It would be good to do a check of all of them as there was some unnecessary logic I removed.